### ISSUE TITLE
[Bug] Can't install backpack pro

### ISSUE THREAD

**Issue by ikblmul** [Laravel-Backpack/CRUD#4619]
# Bug report

So I did install backpack pro with auth.json and get 400 response and then i must insert token

backpack install 1.1.* and not going install

**Comment by pxpm** [comment]
I am sorry, I didn't fully understood your question.
Can you rephrase? 
Post a screenshot or paste the console contents so I can have a look. 

Cheers

**Comment by ikblmul** [comment]
Sorry my bad the password was re generate and the repo backpack has denied last password I use. is the password is regenerate it's self in a periode ?

**Comment by pxpm** [comment]
Hello @ikblmul I don't think the password is automatically regenerated, but you can regenerate it yourself. Probably is what happened here. 

I will be closing this, let me know if you have other issues.

Cheers
